unity-renderer/Assets/DCLPlugins/OutlinerPlugin/Tests/OutlinerControllerShould.cs

```diff
@@ -0,0 +1,32 @@
+﻿// unset:none
+using DCL;
+using NUnit.Framework;
+using System.Linq;
+using System.Reflection;
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+public class OutlinerControllerShould
+{
+    [Test]
+    public void PrepareTheRenderFeatures()
+    {
+        // Arrange
+        var pipeline = ((UniversalRenderPipelineAsset)GraphicsSettings.renderPipelineAsset);
+        FieldInfo propertyInfo = pipeline.GetType().GetField("m_RendererDataList", BindingFlags.Instance | BindingFlags.NonPublic);
```

Comment:
Despite it is a test I would move this reflection stuff to some extensions of sort of as it is a repetitive piece of code

Reply:
Im using Resources.FindObjectsOfTypes as you suggested!